scripts/run-confluece-container.sh

```diff
@@ -33,10 +35,16 @@ function parse_args
     
     # restore positional args
     set -- "${args[@]}"
+    shift 1
 }
 # Set current folder to parent
 cd "$(dirname "$0")"/..
 
+if [[ ! -z "$CONFLUENCE_VERSION" ]]
```

Comment:
While not strictly necessary, is good practice to surround variable with curly braces.
This way variable expand is less error prone, and also is an easy way to see the difference between bash variable and env variable